operator/pkg/controller/karmada/controller.go

```diff
@@ -82,9 +83,11 @@ func (ctrl *Controller) Reconcile(ctx context.Context, req controllerruntime.Req
 
 	klog.V(2).InfoS("Reconciling karmada", "name", req.Name)
 
-	// do reconcile
-
-	return controllerruntime.Result{}, nil
+	planner, err := NewPlannerFor(karmada, ctrl.Client, ctrl.Config)
```

Comment:
Does the `planner` thing is used to install Karmada in parallel?
I think the controller can run in parallel.